src/lammpsio/topology.py

```diff
@@ -414,3 +417,8 @@ def types(self):
     def typeid(self):
         """tuple of int: Type IDs in map."""
         return tuple(self._map.keys())
+
+    @property
+    def inverse(self):
+        """exposes _inverse_map as a read-only property"""
```

Comment:
We have started trying to use type hints. I've also tried to clarify the docstring.
```suggestion
    def inverse(self) -> dict[str, int]:
        """dict: Inverse map from type label to type id."""
```